pages.ko/common/lsof.md

```diff
@@ -0,0 +1,37 @@
+# lsof
+
+> 열린 파일과 상응하는 프로세스들을 나열합니다.
+> 참고: 다른 사람으로부터 열린 파일 리스트는 루트 권한 ( 혹은 sudo ) 이 요구됩니다.
+> 더 많은 정보: <https://manned.org/lsof>.
+
+- 주어진 파일을 열고있는 프로세스 찾기:
+
+`lsof {{파일/의/경로}}`
+
+- 로컬 인터넷 포트를 열고있는 프로세스 찾기:
+
+`lsof -i :{{포트}}`
+
+- 프로세스 아이디 (PID)만 출력:
+
+`lsof -t {{파일/의/경로}}`
+
+- 주어진 유저에 의해 열린 파일 나열:
+
+`lsof -u {{유저이름}}`
+
+- 주어진 명령어 혹은 프로세스에 의해 열린 파일 나열:
+
+`lsof -c {{프로세스_혹은_명령어_이름}}`
+
+- 주어진 PID와 일치하는 프로세스에 의해 열린 파일 나열:
+
+`lsof -p {{PID}}`
+
+- 디렉토리 안의 열린 파일 나열:
+
+`lsof +D {{디렉토리/의/경로}}`
+
+- 로컬 IPv6 TCP 포트에서 수신 대기중인 프로세스 찾기 (네트워크나 포트번호를 변경하지 않음):
```

Comment:
I do not understand the meaning of this part, so I want to confirm

original text: `Find the process that is listening on a local IPv6 TCP port and don't convert network or port numbers`
translated: `Find processes listening on local IPv6 TCP port (without changing network or port number)`

is this translate correct?